husky_control/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ catkin_package()
 
 roslaunch_add_file_check(launch)
 
-catkin_install_python(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(PROGRAMS scripts/stow_ur5 scripts/stow_ur5_left scripts/stow_ur5_right DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
I've added this today after original PR - it's labelled as an odd change; it's for some reason necessary for catkin tools to work correctly - otherwise it doesn't copy the files into the correct install workspace